src/qz/utils/FileUtilities.java

```diff
@@ -310,6 +330,7 @@ public static boolean isGoodExtension(Path path) {
      * e.g. %APPDATA%/qz/data or $HOME/.qz/data, etc
      */
     public static boolean isWhiteListed(Path path, boolean allowRootDir, boolean sandbox, Certificate cert) {
+        // fixme: this is broken
```

Comment:
This also is what determines the demo's folder name since it has no commonname. It should be UNTRUSTED but demoCert.isTrusted() resolves as true as of https://github.com/qzind/tray/commit/f12bfc02b22bda2762a51f817f4426b547ad5d83